scripts/deploy/protocol/04_deployEASResolver.ts

```diff
@@ -63,19 +65,19 @@ task('deployEASResolver', 'Deploys or updates the EAS Resolver and scoutgame att
     console.log('EAS Resolver contract deployed at:', deployedResolver.address);
 
     outputContractAddress({
-      name: 'ProtocolEASResolver',
+      name: 'ScoutProtocolEASResolver',
       address: resolverAddress,
       network: getConnectorKey(connector.chain.id),
-      contractArtifactSource: 'contracts/protocol/contracts/EAS/ProtocolEASResolver.sol:ProtocolEASResolver',
-      deployArgs: deployArgs.slice()
+      contractArtifactSource: 'contracts/protocol/contracts/EAS/ScoutProtocolEASResolver.sol:ScoutProtocolEASResolver',
```

Comment:
If we are changing the contractpath here we should also change in the original file